site/docs/workflows/flutter_pub_publish.md

```diff
@@ -41,18 +43,6 @@ The Flutter Pub Publish workflow consists of the following steps:
 
 **Default** `"ubuntu-latest"`
 
-## Secrets
-
-### `pub_credentials`
```

Comment:
I wouldn't delete this, but instead add a deprecation warning, so that users that are still using can learn about it if needed.

Reply:
I figured that was going to be the feedback. My thinking behind this is we should be pointing people to the new stuff rather than having the old stuff hanging around.